integration/spark/src/test/java/io/openlineage/spark/agent/SerializeOpenLineageEventTest.java

```diff
@@ -0,0 +1,99 @@
+package io.openlineage.spark.agent;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.openlineage.client.OpenLineage;
+import io.openlineage.spark.agent.client.OpenLineageClient;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+
+public class SerializeOpenLineageEventTest {
```

Comment:
I would simplify the class name to just `LineageRunEventTest` as it's assumed this test suite will have tests related to